src/main/java/net/openhft/posix/internal/jnr/JNRPosixAPI.java

```diff
@@ -18,6 +18,7 @@
     static final jnr.ffi.Platform NATIVE_PLATFORM = Platform.getNativePlatform();
     static final String STANDARD_C_LIBRARY_NAME = NATIVE_PLATFORM.getStandardCLibraryName();
     static final Pointer NULL = Pointer.wrap(RUNTIME, 0);
+    static final int SYS_mlock2 = 325; // mlock2 syscall value
```

Comment:
It also looks like syscall number is arch dependent, so we need to make sure to not call it on non-AMD64, such as ARM: https://chromium.googlesource.com/chromiumos/docs/+/master/constants/syscalls.md